### PR TITLE
feature-PRLC100-180 feature changes

### PR DESCRIPTION
**Pull request by niranjan-301**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/steps/c100-rebuild/miam/valid-reason/content.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  en,
+  cy,
+};
+
+export const form: FormContent = {
+  fields: {
+    valid_reason: {
+      type: 'radios',
+      classes: 'govuk-radios',
+      // label: l => l.label,
```
Comment (channa-patil) [inline src/main/steps/c100-rebuild/miam/valid-reason/content.ts:54]: @niranjan-301 Please remove unused commented code line.

File: src/main/steps/c100-rebuild/miam/valid-reason/content.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  two: 'No',
+  errors: {
+    valid_reason: {
+      required: 'Select yes if you have attended a Mediation Information and Assessment Meeting(MIAM)',
+    },
+  },
+});
+
+const cy = () => ({
+  lvTitle: 'Do you have valid reasons for not attending a MIAM? - Welsh',
```
Comment (channa-patil) [inline src/main/steps/c100-rebuild/miam/valid-reason/content.ts:26]: @niranjan-301 Please rename to 'title'

File: src/main/steps/c100-rebuild/miam/valid-reason/content.ts [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+});
+
+const languages = {
+  en,
+  cy,
+};
+
+export const form: FormContent = {
+  fields: {
+    valid_reason: {
```
Comment (channa-patil) [inline src/main/steps/c100-rebuild/miam/valid-reason/content.ts:51]: @niranjan-301 Please follow naming conventions defined for features.
Rename it to 'miam_validReason

File: src/main/steps/c100-rebuild/c100sequence.test.ts [hunk 4]
```diff
@@ -375,5 +375,9 @@ describe('C100Sequence', () => {
     expect(C100Sequence[46].url).toBe('/c100-rebuild/safety-concerns/applicant/concerns-about');
     expect(C100Sequence[46].showInSection).toBe('c100');
     expect(C100Sequence[46].getNextStep({})).toBe('/c100-rebuild/safety-concerns/applicant/concerns-about');
+
+    expect(C100Sequence[47].url).toBe('/c100-rebuild/miam/valid-reason');
+    expect(C100Sequence[47].showInSection).toBe('c100');
+    expect(C100Sequence[47].getNextStep({ valid_reason: YesOrNo.YES })).toBe('/c100-rebuild/miam/valid-reason');
```
Comment (channa-patil) [inline src/main/steps/c100-rebuild/c100sequence.test.ts:381]: @niranjan-301 Can you please add test case for NO flow ?

File: src/main/app/case/case.ts [hunk 5]
```diff
@@ -208,6 +208,7 @@ export interface Case {
   haveSafetyConcerns?: YesOrNo;
   op_courtProceedingsOrders?: C100OrderTypes[];
   op_otherProceedings?: OtherProceedings;
+  valid_reason?: YesOrNo;
```
Comment (channa-patil) [inline src/main/app/case/case.ts:211]: Please rename to miam_validReason

File: src/main/steps/c100-rebuild/miam/valid-reason/content.ts [hunk 6]
```diff
@@ -0,0 +1,82 @@
+import { YesOrNo } from '../../../../app/case/definition';
+import { TranslationFn } from '../../../../app/controller/GetController';
+import { FormContent } from '../../../../app/form/Form';
+import { isFieldFilledIn } from '../../../../app/form/validation';
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+const en = () => ({
+  lvTitle: 'Do you have valid reasons for not attending a MIAM?',
```
Comment (channa-patil) [inline src/main/steps/c100-rebuild/miam/valid-reason/content.ts:8]: @niranjan-301 What's this 'lv' prefix before all these fields ?
Can we please remove it ?

**Review by channa-patil — CHANGES_REQUESTED** [review]

File: src/main/steps/urls.ts [hunk 7]
```diff
@@ -237,3 +237,4 @@ export const C100_DOCUMENT_SUBMISSION: PageLink = `${C100_URL}/document-submissi
 /** @C100 MIAM */
 export const C100_MIAM: PageLink = `${C100_URL}/miam`;
 export const C100_MIAM_MEDIATOR_DOCUMENT: PageLink = `${C100_MIAM}/mediator-document`;
+export const C100_VALID_REASON: PageLink = `${C100_URL}/miam/valid-reason`;
```
Comment (channa-patil) [inline src/main/steps/urls.ts:240]: @niranjan-301 Please use MIAM base URL & then append specific page URL.
`export const C100_MIAM_VALID_REASON: PageLink = `${C100_MIAM}/valid-reason`;`

**Review by channa-patil — CHANGES_REQUESTED** [review]

File: src/main/steps/c100-rebuild/miam/valid-reason/content.ts [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+  lvparagraph1: 'You must attend a MIAM before making an application unless you have valid reasons not to attend.',
+  lvparagraph2: "If you're unsure, you can check the ",
+  applyForLvrLink:
```
Comment (channa-patil) [inline src/main/steps/c100-rebuild/miam/valid-reason/content.ts:11]: @niranjan-301 We can combine lvparagraph2, applyForLvrLink & lvparagraph3 into a single field please.

**Review by channa-patil — CHANGES_REQUESTED** [review]

File: src/main/steps/c100-rebuild/miam/valid-reason/content.ts [hunk 9]
```diff
@@ -0,0 +1,77 @@
+import { YesOrNo } from '../../../../app/case/definition';
+import { TranslationFn } from '../../../../app/controller/GetController';
+import { FormContent } from '../../../../app/form/Form';
+import { isFieldFilledIn } from '../../../../app/form/validation';
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+const en = () => ({
+  title: 'Do you have valid reasons for not attending a MIAM?',
```
Comment (channa-patil) [inline src/main/steps/c100-rebuild/miam/valid-reason/content.ts:8]: @niranjan-301 Can we add service name field as well ?
`serviceName: 'Child arrangements',`

**Review by channa-patil — APPROVED** [review]
LGTM

File: src/main/steps/c100-rebuild/miam/valid-reason/content.ts (hunk 9 above)
Comment (niranjan-301) [reply src/main/steps/c100-rebuild/miam/valid-reason/content.ts:9]: Added serviceName

**Pull request merged**